code/modules/urist/mob/humanhostiles.dm

```diff
@@ -254,7 +254,6 @@
 	faction = "terran"
 
 /mob/living/simple_animal/hostile/urist/terran/marine_space
-	faction = "terran"
```

Comment:
The removed line is an override, think of it as an extra layer on top of a pile that takes precedence above anything below it.

With this removed, we fall back to the parent class (`/mob/living/simple_animal/hostile/urist/terran`), which doesn't set it at all, so we keep going to the grandparent, `/mob/living/simple_animal/hostile/urist`, which sets it to "neutral".

"neutral" makes mobs, well, neutral to typical spessmen, so this should be good. There's some extra special sauce in the Factions subsystem, but by the looks of it it should work as intended too.